app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -259,7 +259,8 @@ class PatientRepository @Inject constructor(
       loggedInUser: User,
       facility: Facility,
       supplyUuidForBpPassport: () -> UUID,
-      supplyUuidForAlternativeId: () -> UUID
+      supplyUuidForAlternativeId: () -> UUID,
+      supplyUuidForPhoneNumber: () -> UUID
```

Comment:
This is the same reason as earlier because these models may or may not get created, so we supply these as functions so they get evaluated lazily.

This is something that will be resolved when we pick up https://www.pivotaltracker.com/story/show/173122237.